source/code/source/playfab/PlayFabCallRequestContainer.cpp

```diff
@@ -32,11 +32,16 @@ namespace PlayFab
         const char* reqBod = requestBody.c_str();
         size_t reqBodLength = requestBody.length();
 
-        bool parsingSuccessful = reader->parse(reqBod, reqBod + reqBodLength, &request, &errs);
-
-        if (parsingSuccessful)
+        try {
+            bool parsingSuccessful = reader->parse(reqBod, reqBod + reqBodLength, &request, &errs);
+            if (parsingSuccessful)
+            {
+                errorWrapper.Request = request;
+            }
+        }
+        catch (std::exception)
         {
-            errorWrapper.Request = request;
+            // We can't parse the request back into a JSON::Value, so the caller won't receive it back...?
```

Comment:
Did you mean for this to sound ambiguous?

Reply:
Yes?
I tried to make it declare this call invalid/failure here, but that didn't work.
Essentially, it's a bug that we hadn't seen before, and resolving it properly wasn't trivial.
Good call though, I'll file a bug.

Reply:
Bug filed.